tools/schemacode/src/bidsschematools/render/tsv.py

```diff
@@ -1,15 +1,57 @@
+from __future__ import annotations
+
 import io
+import re
 
 import pandas as pd
 from markdown_it import MarkdownIt
 from tabulate import tabulate
 
 
-def fence(source: str, language: str, css_class: str, options: dict, md, **kwargs) -> str:
+def fence(
+    source: str,
+    language: str,
+    css_class: str,
+    options: dict,
+    md,
+    classes=None,
```

Comment:
If `classes` is never none, then we should just use:

```suggestion
    classes: list[str],
```